.github/workflows/latest-install-test-react-native.yml

```diff
@@ -0,0 +1,97 @@
+# This workflow installs
+# - the latest Realm JS,
+# - with latest Node.js & NPM into
+# - a newly initialized React Native app (running the latest version of React Native)
+
+name: Latest install test (React Native)
+
+on:
+  # Every Monday at 9:00 CET
+  schedule:
+    - cron: "0 8 * * 1"
+  # You can also activate this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  install:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 'latest'
+      - name: Initialize a React Native app
+        run: npx react-native init ReactNativeTestApp --directory react-native-test-app --npm
+      - name: Install Realm JS
+        working-directory: react-native-test-app
+        run: npm install realm
+      - name: Install CocoaPods
+        working-directory: react-native-test-app/ios
+        run: pod install
+      - name: Make the app load Realm
+        working-directory: react-native-test-app
+        run: |
+          cat > App.js <<EOL
+            import React, {useEffect} from 'react';
```

Comment:
I'm thinking of alternatives to "cat-ing" the source scripts into the project.  It would be better for maintenance if we had a separate project where this was loaded from so that if for some reason this fails, we have an easy way to fix it.

Reply:
@kraenhansen I think App.js should be a seperate file somewhere so we can easily debug problems.

Reply:
oh sorry, I somehow I was being shown a previous commit in the PR view.